support/Pipelines/EccentricityControl/EccentricityControl.py

```diff
@@ -290,8 +290,8 @@ def coordinate_separation_eccentricity_control_digest(
             ]
             logger.info("Updated Xcts values based on fit:")
             logger.info(
-                f"(Omega, adot) = ({(xcts_omega + dOmg):13.10f},"
-                f" {(xcts_expansion + dadot):13.10g})"
+                f"(Omega, adot) = ({(xcts_omega):13.10f},"
+                f" {(xcts_expansion):13.10g})"
```

Comment:
Just for clarity, `xcts_omega` and `xcts_expansion` already hold the updated values which is why you don't need to add the change in omega/expansion?

Reply:
Yes, I had forgot to update the print statement after I added the updated values to the dictionary.

Reply:
Ok thanks!